micro-ui/web/micro-ui-internals/packages/modules/workbench/src/utils/index.js

```
@@ -191,17 +191,26 @@
   }
   switch (type) {
     case "date":
-      return new Date(value)?.toISOString?.()?.split?.("T")?.[0];
+      const options = { timeZone: 'Asia/Kolkata', year: 'numeric', month: '2-digit', day: '2-digit' };
+      const indianDate = new Date(value).toLocaleDateString('en-CA', options); 
+      return indianDate;
     case "datetime":
       return new Date(value).toISOString();
     case "EPOC":
-      return String(new Date(value)?.getTime());
+      const date = new Date(value);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)